RFC.md

```diff
@@ -179,6 +182,14 @@ enum class UpdateStrategy {
     AlwaysUpdate,
     OnlyFetchOnce,
 }
+
+enum class ReadingDirection {
+    LeftToRight,
+    RightToLeft,
+    Vertical,
+    LongStrip,
+    LongStripWithGaps,
```

Comment:
Other 2 are more or less for users to select
```suggestion
    LeftToRight,
    RightToLeft,
    LongStrip,
```

Reply:
I kinda wanted them there for user controlled extensions I guess, like self hosted ones, but makes sense to trim them.